waves/src/WalletInfo.tsx

```diff
@@ -39,7 +39,7 @@ export default function WalletInfo({ balance }: WalletInfoProps) {
         console.log("Withdrawing to " + withDrawAddress);
     };
 
-    async function fundWallet(): Promise<any> {
+    async function fundLbtc(): Promise<any> {
```

Comment:
I think we should just unify these.

Bobtimus can easily fund both of them at the same time :)
Maybe even with the same button / endpoint?